wandb/plot/line_series.py

```diff
@@ -67,14 +70,14 @@ def line_series(
         xs = [xs for _ in range(len(ys))]
     assert len(xs) == len(ys), "Number of x-lines and y-lines must match"
 
-    if keys is not None:
-        assert len(keys) == len(ys), "Number of keys and y-lines must match"
-
-    data = [
-        [x, f"key_{i}" if keys is None else keys[i], y]
-        for i, (xx, yy) in enumerate(zip(xs, ys))
-        for x, y in zip(xx, yy)
-    ]
+    if log_scale:
```

Comment:
you removed the 
if keys is not None:
assert len(keys) == len(ys), "Number of keys and y-lines must match"